process/project-lifecycle-documents/openssf_scorecard_incubating_stage.md

```diff
@@ -0,0 +1,89 @@
+# OpenSSF Scorecard — Incubating application
+
+## Project has met all Sandbox requirements
+
+The only Sandbox application requirement that is not listed as part of the Incubation application superset is the matter of project sponsorship.
+
+### Sponsor
+
+Most projects will report to an existing OpenSSF Working Group, although in some cases a project may report directly to the TAC. The project commits to providing quarterly updates on progress to the group they report to.
+
+OpenSSF Scorecard is a project of the Best Practices Working Group.
+
+## List of project maintainers
+
+The project must have a minimum of three maintainers with a minimum of two different organizational affiliations.
+
+- Stephen Augustus, Cisco, [@justaugustus](https://github.com/justaugustus)
+- Raghav Kaul, Google, [@raghavkaul](https://github.com/raghavkaul)
+- Jeff Mendoza, Kusari, [@jeffmendoza](https://github.com/jeffmendoza)
+- Spencer Schrock, Google, [@spencerschrock](https://github.com/spencerschrock)
+- Laurent Simon, Independent, [@laurentsimon](https://github.com/laurentsimon)
```

Comment:
I thought Laurent was still at Google, is that really accurate?

Reply:
@laurentsimon